### PR TITLE
fix: upgrade fastchat to v0.2.36

### PR DESCRIPTION
**Pull request by 0xff-dev**
#### What type of PR is this?
/kind feature

#### What this PR does / why we need it
fix: upgrade fastchat to v0.2.36
#### Which issue(s) this PR fixes

Fix https://github.com/kubeagi/arcadia/issues/748

### CODE REVIEW

**Comment by bjwswang** [comment]
test failed

**Review by Abirdcfly — APPROVED** [review]
LGTM, let us wait test to pass.

**Comment by Abirdcfly** [comment]
https://github.com/marketplace/actions/maximize-build-disk-space

File: deploy/llms/Dockerfile.fastchat-worker [hunk 1]
```diff
@@ -1,34 +1,44 @@
...
+
+# Official: https://pypi.org/simple
+ARG PYTHON_INDEX_URL="https://pypi.mirrors.ustc.edu.cn/simple/"
+
+# Install fastchat along with its dependencies
+RUN apt-get install -y python3.9 python3.9-distutils curl python3-pip python3-dev
+RUN python3.9 -m pip install tomli setuptools_scm wavedrom -i ${PYTHON_INDEX_URL}
+RUN python3.9 -m pip install --upgrade pip -i ${PYTHON_INDEX_URL}
+#RUN python3.9 -m pip install fschat==v0.2.36 fschat[model_worker]==v0.2.36 pydantic==1.10.14 -i ${PYTHON_INDEX_URL}
+RUN git clone https://github.com/lm-sys/FastChat.git \
```
Comment (Lanture1064) [inline deploy/llms/Dockerfile.fastchat-worker:27]: Install git before git cloning

**Review by Lanture1064 — APPROVED** [review]

**Review by bjwswang — APPROVED** [review]

**Pull request merged**